src/db.c

```diff
@@ -2004,6 +2027,31 @@ db_files_get_count_bymatch(char *path)
 #undef Q_TMPL
 }
 
+int
+db_file_get_seekpos(uint32_t id)
```

Comment:
I do not have a strong opinion on this and will change it if you want, but it results in a (probably small) performance decrease, as fetching the whole media_file_info makes memory allocations for every string field in media_file_info.

I did consider adding it to the queue item, but i do not believe it is the right place (would require updating the queue by the player and there maybe places where it is not desirable to store it in the queue - e. g. if it is not the item where playback starts).